spec/requests/api/v1/merchant_request_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe "Merchants" do
+  it "gets a list of all merchants" do
+    Merchant.create!(name: "Walmart")
+    Merchant.create!(name: "Target")
+    Merchant.create!(name: "Sam's")
+
+    get "/api/v1/merchants"
+
+    expect(response).to be_successful
+
+    merchants = JSON.parse(response.body, symbolize_names: true)
+    expect(merchants[:data].count).to eq(3)
```

Comment:
Excellent use of symbolize_names! Glad we created our own problem and solved it.